databuilder/databuilder/task/search/search_data_queries.py

```diff
@@ -11,82 +11,76 @@
     MATCH (db:Database)<-[:CLUSTER_OF]-(cluster:Cluster)
     <-[:SCHEMA_OF]-(schema:Schema)<-[:TABLE_OF]-(table:Table)
     {publish_tag_filter}
-    OPTIONAL MATCH (table)-[:DESCRIPTION]->(table_description:Description)
     OPTIONAL MATCH (schema)-[:DESCRIPTION]->(schema_description:Description)
-    OPTIONAL MATCH (table)-[:DESCRIPTION]->(prog_descs:Programmatic_Description)
-    WITH db, cluster, schema, schema_description, table, table_description,
-    COLLECT(prog_descs.description) as programmatic_descriptions
+    OPTIONAL MATCH (table)-[:DESCRIPTION]->(table_description:Description)
     OPTIONAL MATCH (table)-[:TAGGED_BY]->(tags:Tag) WHERE tags.tag_type='default'
-    WITH db, cluster, schema, schema_description, table, table_description, programmatic_descriptions,
```

Comment:
is it intended to remove programmatic_descriptions from search?

Reply:
Yes, if you look at search right now it doesn't actually use programmatic descriptions. If needed it can be added by configuring the query with custom values instead of default values of the variables within the query.